packages/snaps-execution-environments/jest.config.js

```diff
@@ -9,7 +9,11 @@ module.exports = deepmerge(baseConfig, {
   coverageProvider: 'v8',
   projects: [
     deepmerge(baseConfig, {
-      coveragePathIgnorePatterns: ['index.ts', '.ava.test.ts'],
+      coveragePathIgnorePatterns: [
+        'index.ts',
+        '.ava.test.ts',
+        '<rootDir>/src/common/test-utils/testUtils.ts',
```

Comment:
Can we just ignore the entire `test-utils` folder, in the same way that is done in other projects?

Reply:
Updated config to ignore entire test-utils folder now.